### PR TITLE
fix wildcard comments

### PR DESCRIPTION
**Pull request by hzxuzhonghu**
**Please provide a description of this PR:**

### CODE REVIEW

**Comment by hzxuzhonghu** [comment]
cc @wulianglongrd

File: pilot/pkg/networking/core/v1alpha3/route/route.go [hunk 1]
```diff
@@ -1536,6 +1536,8 @@ func SortVHostRoutes(routes []*route.Route) []*route.Route {
 // isCatchAllRoute returns true if an Envoy route is a catchall route otherwise false.
 func isCatchAllRoute(r *route.Route) bool {
 	catchall := false
+	// A Match is catch all if and only if it has no header/query param match
+	// and URI has a prefix / or regex .*.
```
Comment (wulianglongrd) [inline pilot/pkg/networking/core/v1alpha3/route/route.go:1540]: I think it would be better to omit the last `.`, or add a backtick.

Comment (wulianglongrd) [reply pilot/pkg/networking/core/v1alpha3/route/route.go:1540]: Although I think it looks ok this way.

Comment (zirain) [inline pilot/pkg/networking/core/v1alpha3/route/route.go:1540]:
```suggestion
	// and URI has a prefix `/` or regex `.*`.
```

Comment (hzxuzhonghu) [reply pilot/pkg/networking/core/v1alpha3/route/route.go]: HHA

**Review by ramaraochavali — APPROVED** [review]

**Pull request merged**